projects/questionset-editor-library/src/lib/services/editor/editor.service.spec.data.ts

```diff
@@ -275,6 +268,7 @@ export const questionsData = [
         channel: '01309282781705830427',
         responseDeclaration: {
             response1: {
+                maxScore: 1,
```

Comment:
@ramkumar-pacewisdom As per the new format we cannot store the maxScore property inside the responseDeclaration. Please check this document https://project-sunbird.atlassian.net/wiki/spaces/QB/pages/3317628929/QuML+compliance+implementation+changes+for+editor+and+player#maxScore